source/index.txt

```diff
@@ -130,15 +180,16 @@ MongoDB University
 
 MongoDB University provides free courses to teach everyone how to use MongoDB.
 
-Take the free online course taught by MongoDB instructors
-`````````````````````````````````````````````````````````
+Take the following free online courses taught by MongoDB instructors:
+`````````````````````````````````````````````````````````````````````
 
 .. list-table::
-
-   * - .. figure:: /includes/figures/M220JS_hero.jpg
-          :alt: Banner for the MongoDB University Node.js Course
      
```

Comment:
I ended up taking both pictures out since it looked kinda weird with just one, but let me know if you think I should keep the original graphic there